### ISSUE TITLE
Issue: PlatformNotSupportedException

### ISSUE THREAD

**Issue by alexangas** [WireMock-Net/WireMock.Net#104]
Hello,

Just tried creating a new console app with WireMock 1.3.4 on .NET Framework 4.6.1 using lowest NuGet dependencies. Unfortunately when running with the following code...

```csharp
var server = FluentMockServer.Start();
server.Given(
        Request.Create()
    )
    .RespondWith(
        Response.Create().WithProxy("http://10.10.66.65")
    );
```

... this exception occurs:

```
System.PlatformNotSupportedException: Operation is not supported on this platform.
   at WireMock.Http.HttpClientHelper.CreateHttpClient(String clientX509Certificate2ThumbprintOrSubjectName)
   at WireMock.ResponseBuilders.Response.WithProxy(String proxyUrl, String clientX509Certificate2ThumbprintOrSubjectName)
   at WireMockNet461.Program.Main(String[] args) in C:\Users\alexangas\source\repos\WireMockNet_461\WireMockNet_461\Program.cs:line 17
```

I've tried every version of .NET from 4.6.1 to 4.7.1 and they have the same issue. Cannot reproduce with .NET Core. Also have tried updating all of the NuGet packages. Last version of WireMock I was using successfully was 1.0.2.11.

**Comment by StefH** [comment]
I cannot reproduce, can you take a look at the new example (WireMock.Net.StandAlone.Net461) which uses the NuGet?

**Comment by alexangas** [comment]
I'm away from a machine right now but noticed browsing the linked commit that it's targeting 4.5.2. I didn’t try below 4.6.1 - maybe it works in that version?

**Comment by StefH** [comment]
Indeed, I get same error. I'll investigate.

**Comment by StefH** [comment]
Will be solved in 1.0.3.7
